src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -19,23 +19,65 @@ import '../TextBlock/TextBlock';
 import '../TextRun/TextRun';
 
 /**
- * AdaptiveCard component
+ * AdaptiveCardInternal component
  *
  * @param {object} props  React properties
  * @param {object} props.data  Active Cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
  * @returns {object} JSX of the component
  */
-export default function AdaptiveCard({data}) {
-  const [cssClasses] = webexComponentClasses('adaptive-card', 'wxc-ac-container--has-padding wxc-ac-container--vertical');
+function AdaptiveCardInternal({data, className, style}) {
+  const [cssClasses] = webexComponentClasses('adaptive-card', className);
 
   return (
-    <div className={cssClasses}>
+    <div className={`${cssClasses} wxc-ac-container--has-padding`} style={style}>
       {/* eslint-disable react/no-array-index-key */}
       {data.body.map((item, index) => <Component data={item} key={index} parentData={data} />)}
     </div>
   );
 }
 
+/**
+ * AdaptiveCard component
+ *
+ * @param {object} props  React properties
+ * @param {object} props.data  Active Cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function AdaptiveCard({data, className, style}) {
+  return <Component data={data} className={className} style={style} />;
+}
+
 AdaptiveCard.propTypes = {
   data: PropTypes.shape().isRequired,
+  className: PropTypes.string,
+  style: PropTypes.shape(),
 };
+
+AdaptiveCard.defaultProps = {
+  className: '',
+  style: undefined,
+};
+
+AdaptiveCardInternal.propTypes = {
```

Comment:
please move all the lines from here to the end of the file just before the `AdaptiveCard` function definition, so they are together with their component